src/test/java/pages/ProfilePage.java

```diff
@@ -0,0 +1,50 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import java.util.UUID;
+
+public class ProfilePage extends BasePage{
+
+
+    public String getProfileName() {
+        WebElement profileName = getDriver().findElement(By.cssSelector("span.name"));
```

Comment:
We can make the xpath or css the class variable (Locator)

For example

   @FindBy(css = "img.avatar")
    private WebElement avatar
```